packages/uploadthing/src/internal/handler.ts

```diff
@@ -128,17 +133,20 @@ export type UploadThingResponse = {
   chunkSize: number;
 }[];
 
-export const buildRequestHandler = <TRouter extends FileRouter>(
+export const buildRequestHandler = <
+  TRouter extends FileRouter,
+  Args extends MiddlewareFnArgs<any, any, any>,
```

Comment:
added the generic here for extra typesafety and prevent future regressions when adding new adapters etc. now it must match what the adapter has put in their `createUploadthing()` generic